cargo-paradedb/Cargo.toml

```diff
@@ -4,14 +4,22 @@ version = "0.10.1"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
```

Comment:
@neilyio should review here. I believe he intentionally made `cargo-paradedb` be Postgres-agnostic. We may want to preserve that if we can.